database/index.js

```diff
@@ -0,0 +1,30 @@
+const mysql = require('mysql');
+const mysqlConfig = require('./config.js');
+
+const connection = mysql.createConnection(mysqlConfig);
+
+const getAllProducts = function (callback) {
+    var query = 'select * from products';
+    connection.query(query, function (err, results) {
+      if (err) {
+        console.log(err)
+      }
+      callback(err, results);
+    });
+  }
+
+  const postAllProducts = function (product , callback) {
+      var query = `INSERT INTO products (title, price, description, category, image) VALUES ('${product.title}', ${product.price}, '${product.description}', '${product.category}', '${product.image}')`
```

Comment:
using this method leaves you open to scripting attacks. consider using a queryArgs var equal to the array of values you have. and replacing each thing with a ? mark instead.